data/sssd-tests/local/test.sh

```diff
@@ -3,6 +3,23 @@
 set -e
 
 . ../testincl.sh
+
+isSles15(){
+   if [ ! -f /etc/os-release ]; then
+      return 1
+   fi
+   if (grep -i -q "SUSE Linux Enterprise Server 15" /etc/os-release); then
```

Comment:
why not source and use $VERSION_ID variable instead?

Reply:
simply ease of implementation, there are probably many different ways to detect the version, I saw elsewhere code doing this

``` bash
isSles15(){
   if [ ! -f /etc/os-release ]; then
      return 1
   fi
   .  /etc/os-release

   if [[ "$VERSION_ID" == "15"* ]]; then
      return 0
   fi
   return 1
}
```
should work too, I actually prefer the simple grep (but I am not in anyway comfortable writing sh scripts :-)) Please let me know if you wish me to update with the change above or if the orig code is ok

Reply:
Don't worry @noelpower :), was more out of curiosity rather than being picky

Reply:
thanks alot!!